### DISCUSSION TITLE
Q-Table component has an issue in initialization.

### DISCUSSION THREAD

**Discussion by Claus1** [quasarframework/quasar#11279] [Ideas / Proposals]
My custom table uses Q-Table and on load I have to load 'selected' property from my data.
I make computed rows() which makes rows from my data. And I have to initialize 'selected' property from the 'rows', because q-table requests its objects. If it works with key it would be normal, but how to make it without conflict?
Ok, do it in mounted(). I have a watcher for selected and such init in mounted() causes watch handler, but 'selected' value  did not changed. It's init value . So if you see good way to get round the issue, I will glad to hear)

**Comment by rstoenescu** [comment]
Hi,

Attaching a [codepen](https://codepen.quasar.dev) or [codesandbox](https://codesandbox.quasar.dev) link of your scenario will be infinitely more helpful for people to offer assistance.

**Comment by Claus1** [comment]
Thanks! I found the solution with Composition api, with clear Option API it was impossible.
